server/main.go

```diff
@@ -41,6 +44,10 @@ func LoggingMiddleware(next http.Handler) http.Handler {
 
 func main() {
 	gotenv.Load()
+	if os.Getenv("CLERK_SECRET_KEY") == "" {
+		panic("CLERK_SECRET_KEY is not set")
```

Comment:
could be nice to allow the backend to run without the secret by not registering the protected routes, so people can still work on other parts of the backend